src/Basisregisters.IntegrationDb.SuspiciousCases/Migrations/20240503095640_Initial.cs

```diff
@@ -30,7 +30,10 @@ protected override void Up(MigrationBuilder migrationBuilder)
             migrationBuilder.Sql(CurrentAddressLinkedWithBuildingUnitButNotWithParcelConfiguration.Create);
             migrationBuilder.Sql(CurrentAddressWithoutLinkedParcelOrBuildingUnitConfiguration.Create);
             migrationBuilder.Sql(CurrentAddressWithSpecificationDerivedFromBuildingUnitWithoutLinkedBuildingUnitConfiguration.Create);
+            migrationBuilder.Sql(CurrentStreetNameWithoutLinkedRoadSegmentsConfiguration.Create);
```

Comment:
@ArneD I've extended the existing migration. Or do you prefer a new one?

Reply:
The reason why is because migrationBuilder.Sql(SuspiciousCaseCountConfiguration.Create); on line 39 is also changed under the hood.

Reply:
You'd need a new one else it won't get executed.